### PR TITLE
Update ap-base packages with 3.18.8

### PR DESCRIPTION
**Pull request by himabindu07**
## Details

Update ap-base packages with 3.18.8

Image Name |  old  Tag| New tag |
-- | -- | -- |
ap-blackbox-exporter | 0.25.0-1| 0.25.0-2
ap-registry| 3.18.7-1 | 3.18.8
ap-postgres-exporter|0.15.0-6 |  0.15.0-7
ap-nats-streaming| 0.25.6-4 | 0.25.6-5
ap-nats-server| 2.10.14-1 | 2.10.14-2
ap-nats-exporter| 0.15.0-1 | 0.15.0-2
ap-init | 3.18.7-1 | 3.18.8
ap-curator | 8.0.15-1 |  8.0.16-1
ap-awsesproxy| 1.5.0-9 | 1.5.0-10
git-daemon  | 3.18.7-1 | 3.18.8
git-sync-relay | 0.0.3-4 |0.0.3-5
ap-pgbouncer-exporter | 0.15.0-9 | 0.15.0-10
ap-vector | 0.48.0| 0.48.0-1
ap-pgbouncer-krb |1.17.0-12 | 1.17.0-13
ap-pgbouncer | 1.23.1 | 1.23.1-1
ap-base | 3.18.7-1 | 3.18.3 

## Related Issues

https://github.com/astronomer/issues/issues/6557

## Checklist

- [X] version.txt was updated

### CODE REVIEW

**Comment by Simpcyclassy** [comment]
Would the pipeline job failing on this PR be fixed by these changes?

**Review by danielhoherd — APPROVED** [review]
LGTM

File: nats-streaming/twistcliignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#New CVEs that are not found in nats-streaming/twistcliignore: 
+CVE-2023-45288
+CVE-2024-24790
+
+#Old CVEs that exist now but are already in nats-streaming/twistcliignore: 
 CVE-2023-39326
```
Comment (danielhoherd) [inline nats-streaming/twistcliignore:6]:
```suggestion
CVE-2023-45288
CVE-2024-24790
```

File: registry/twistcliignore [hunk 2]
```diff
@@ -1,12 +1,42 @@
...
+CVE-2023-24540
+CVE-2023-29400
+CVE-2023-29403
+CVE-2023-29406
+CVE-2023-29409
+CVE-2023-39318
+CVE-2023-39319
+CVE-2023-39326
+CVE-2023-45284
+#Old CVEs that exist now but are already in registry/twistcliignore:
```
Comment (danielhoherd) [inline registry/twistcliignore:32]: This comment means you should remove the CVEs listed below.

**Review by danielhoherd — APPROVED** [review]
The CVE exclude updates look good.

**Comment by himabindu07** [comment]
https://github.com/astronomer/issues/issues/6500

**Pull request merged**